docs/analysis/ue5.8-hzb-occlusion-analysis.md

```
@@ -0,0 +1,328 @@
+if (ShouldRenderNanite() || FInstanceCullingContext::IsOcclusionCullingEnabled())
+    GraphBuilder.QueueTextureExtraction(FurthestHZBTexture, &View.ViewState->PrevFrameViewInfo.HZB);
+```
+
+So the cross-frame loop is **Frame N depth → Frame N HZB → stored → Frame N+1
+main-pass occlusion test.** `RenderHzb` runs *after* `RenderNanite` so the scene HZB
+includes Nanite occluders. Note: this scene-level HZB (furthest-only, from resolved
+depth) is **distinct** from the intra-frame `Nanite.PreviousOccluderHZB` that phase 2
+builds from the VisBuffer — same `BuildHZB` code, different source & lifetime. Async
+build is gated by `r.SceneDepthHZBAsyncCompute`.
+
+### First frame / camera cuts
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)